lib/pliny/commands/updater.rb

```diff
@@ -1,10 +1,11 @@
 require 'fileutils'
 require 'pathname'
 require 'pliny/version'
+require 'pliny/commands/creator'
 require 'uri'
 
 module Pliny::Commands
-  class Updater
+  class Updater < Creator
```

Comment:
Inheriting from the `Creator` looks a little strange from me. Was there any specific decision to do this, other than avoid duplicating a few methods? My worry is that there might be unintended side effects by this if something changes in the `Creator`.